### ISSUE TITLE
Update notification repeatedly appearing

### ISSUE THREAD

**Issue by LXGaming** [ChrisRM/material-theme-jetbrains#1791]
### Summary
The update notification is being displayed every time a project is opened, It's expected that this notification is only shown once per update.
I believe the issue may be due to the version in the config not being updated when the `Show What's new window on updates` option is disabled.

### Environment
- Windows 10
- IntelliJ IDEA 2021.1.3
- Material Theme v6.7.0

```
<application>
  <component name="MaterialThemeConfig">
    <option name="compactMenus" value="true" />
    <option name="highContrast" value="true" />
    <option name="accentScrollbars" value="false" />
    <option name="isCompactMenus" value="true" />
    <option name="isHighContrast" value="true" />
    <option name="isWizardShown" value="true" />
    <option name="overrideAccentColor" value="false" />
    <option name="pristineConfig" value="false" />
    <option name="statusBarTheme" value="false" />
    <option name="upperCaseButtons" value="false" />
    <option name="settingsSelectedTab" value="5" />
    <option name="accentColor" value="2979ff" />
    <option name="highlightColor" value="2979ff" />
    <option name="selectedTheme" value="DARKER" />
    <option name="version" value="6.4.2.1" />
    <option name="showWhatsNew" value="false" />
  </component>
</application>
```

![Capture](https://user-images.githubusercontent.com/10443061/124543030-9e682800-de78-11eb-8bdf-93c866efe5e5.PNG)

**Comment by mallowigi** [comment]
fixing that in a jiffy

**Comment by LXGaming** [comment]
Confirmed fixed in v6.7.2, Thanks for the quick fix.
